src/rez/utils/execution.py

```diff
@@ -73,8 +73,8 @@ def __init__(self, args, **kwargs):
         #
         # NOTE: currently no solution for `python3+<3.6`
         #
-        if sys.version_info[:2] >= (3, 6) and "encoding" in kwargs:
-            kwargs['encoding'] = 'utf-8'
+        if sys.version_info[:2] < (3, 6) and "encoding" in kwargs:
+            kwargs.pop("encoding")
```

Comment:
I'm not sure this is right. Looking at the original code, I think there may have been an existing bug, and L76 should be:

```
if sys.version_info[:2] >= (3, 6) and "encoding" not in kwargs:
```

Reply:
Ah, my original intention was to not pin the encoding to "utf-8" (no specific reason), only removes it when the Python version is not compat, since `encoding` kwarg has been explicit set in commit d38e873, which should be removed if Python version incompat.

Now after #961, I think commit d38e873 should be reverted or it would be Python 2.7 incompat ?